src/cbor/test_decoder.rs

```diff
@@ -366,3 +366,20 @@ fn test_signed_integer_too_large() {
     let bytes = vec![0x3b, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff];
     test_decoder_error(bytes, CborError::InputValueOutOfRange);
 }
+
+#[test]
+fn test_large_input() {
+    let array = vec![0xFF; MAX_ARRAY_SIZE];
+    let expected = CborType::Bytes(array.clone());
+    let mut bytes = vec![0x5A, 0x08, 0x00, 0x00, 0x00];
+    bytes.extend_from_slice(&array);
+    test_decoder(bytes, expected);
+}
+
+#[test]
+fn test_too_large_input() {
+    let array = vec![0xFF; MAX_ARRAY_SIZE+1];
```

Comment:
Huh - I thought rust was opinionated about spaces around operators, but I guess not. In any case, I think we most often include the spaces, so it would be nice to be consistent here.

Reply:
Will fix before merging.